src/test/java/wooteco/subway/controller/SectionControllerTest.java

```diff
@@ -0,0 +1,105 @@
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.jdbc.Sql;
+import wooteco.subway.dao.LineDao;
+import wooteco.subway.dao.SectionDao;
+import wooteco.subway.dao.StationDao;
+import wooteco.subway.domain.Line;
+import wooteco.subway.domain.Section;
+import wooteco.subway.domain.Station;
+import wooteco.subway.domain.entity.LineEntity;
+import wooteco.subway.domain.entity.SectionEntity;
+import wooteco.subway.dto.LineRequest;
+import wooteco.subway.dto.LineResponse;
+import wooteco.subway.dto.SectionRequest;
+import wooteco.subway.service.LineService;
+import wooteco.subway.service.SectionService;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureTestDatabase
+@Sql("classpath:test-schema.sql")
+class SectionControllerTest {
+
+    @LocalServerPort
+    int port;
+
+    @Autowired
+    JdbcTemplate jdbcTemplate;
+
+    SectionDao sectionDao;
+    SectionService sectionService;
+    LineService lineService;
+
+    Station testStation1;
+    Station testStation2;
+    Station testStation3;
+    Station testStation4;
+    Station testStation5;
+    Station testStation6;
+
+    @BeforeEach
+    public void setUp() {
+        RestAssured.port = port;
+        StationDao stationDao = new StationDao(jdbcTemplate);
+        SectionDao sectionDao = new SectionDao(jdbcTemplate);
+        LineDao lineDao = new LineDao(jdbcTemplate);
+        sectionService = new SectionService(sectionDao, stationDao, lineDao);
+        lineService = new LineService(lineDao, sectionDao, stationDao);
+
+        testStation1 = stationDao.save(new Station("testStation1"));
+        testStation2 = stationDao.save(new Station("testStation2"));
+        testStation3 = stationDao.save(new Station("testStation3"));
+        testStation4 = stationDao.save(new Station("testStation4"));
+        testStation5 = stationDao.save(new Station("testStation5"));
+        testStation6 = stationDao.save(new Station("testStation6"));
+
+        LineResponse lineResponse = lineService
+                .createLineAndRegisterSection(new LineRequest("testLine", "color", testStation3.getId(), testStation4.getId(), 10L));
+
+        sectionService.insertSection(lineResponse.getId(), new SectionRequest(testStation4.getId(), testStation5.getId(), 20L));
+    }
+
+    @DisplayName("구간을 저장한다")
+    @Test
+    void createSection_success() {
+        SectionRequest sectionRequest = new SectionRequest(2L, 3L, 10L);
+        ExtractableResponse<Response> extract = RestAssured.
```

Comment:
```suggestion
        RestAssured.
```
변수로 할당하지 않아도 될 것 같아요~

Reply:
변수 할당 하지 않아도 되는 것들 삭제하였습니다!